Python/Product/PythonTools/visualstudio_py_debugger.py

```diff
@@ -2318,7 +2318,7 @@ def detach_threads():
 
 def new_thread(tid = None, set_break = False, frame = None):
     # called during attach w/ a thread ID provided.
-    if tid == debugger_thread_id:
+    if tid and tid == debugger_thread_id:
```

Comment:
Did you see my comment on visualstudio_py_debugger.py? I wonder how to proceed there.